tools/metrics/gauges.go

```diff
@@ -274,3 +312,38 @@ func findService(services []cfclient.Service, guid string) *cfclient.Service {
 	}
 	return nil
 }
+
+func findServicePlan(service_plans []cfclient.ServicePlan, guid string) *cfclient.ServicePlan {
+	for _, service_plan := range service_plans {
+		if service_plan.Guid == guid {
+			return &service_plan
+		}
+	}
+	return nil
+}
+
+func findOrgQuotaFromSpaceGUID(c *Client, guid string) (*cfclient.OrgQuota, error) {
```

Comment:
I think it would be a good idea to cache the results of this method to decrease the number of CF api requests.

Reply:
I agree, but it might make cleaner code to iterate over spaces than to cache this. What do you think?

Reply:
I'm happy with both, but I think your solution is better as caching would increase the complexity.

Reply:
Unsure how to make this pleasant code in Go, but this was a very good call.